client/server/boot/index.js

```diff
@@ -31,6 +32,15 @@ export default function setup() {
 	if ( 'development' === process.env.NODE_ENV ) {
 		require( 'calypso/server/bundler' )( app );
 
+		// When mocking WordPress.com to point locally, wordpress.com/wp-login.php will hit Calypso.
+		// redirect traffic to de.wordpress.com to hit the real backend.
+		if ( process.env.MOCK_WORDPRESSDOTCOM === '1' ) {
+			app.use(
+				'/wp-login.php',
```

Comment:
Should we also handle attempts to go to `/log-in`?

Reply:
/log-in works great.